lib/my_tools/place_data_scraper.rb

```diff
@@ -9,6 +9,10 @@ def initialize(url)
 
     def save_review(place_id)
       results = @scrape_process.fetch_reviews
+      reviews = Review.find_by(place_id: place_id)
+
+      #place_idで探してきたReviewのレコードが既に存在したら既存のレコードを削除する
```

Comment:
```suggestion
      # place_idで探してきたReviewのレコードが既に存在したら既存のレコードを削除する
```

コメントの `#` と実際のコメントの間はスペースあけるようにしましょう